### PR TITLE
fix observation in the code review

### PR DESCRIPTION
**Pull request by robertoaban**
Please take a look
@jmaguina1202 @carledriss @mgvirtual

### CODE REVIEW

File: src/test/java/steps/ResourcesSteps.java [hunk 1]
```diff
@@ -1,38 +1,41 @@
...
 
-        CommonUtils util = new CommonUtils();
+    @When("^I did a post request:$")
+    public void iDidPostRequest(DataTable table) throws
+            Throwable {
+       List<List<String>> data =table.raw();
+       JSONObject project = new JSONObject();
+        for ( int index = 0; index < data.size(); index ++ ) {
+            project.put(data.get(index).get(0), data.get(index).get(1));
+        }
```
Comment (carledriss) [inline src/test/java/steps/ResourcesSteps.java]: You can use directly List<Map<String, String>> as parameter

``` java
public void iDidPostRequest(List<Map<String, String>> table)
```

File: src/test/java/steps/ResponsesSteps.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+public class ResponsesSteps {
+
+    private ResourcesSteps projectSteps;
+
+    public ResponsesSteps(ResourcesSteps projectSteps) {
+       this.projectSteps = projectSteps;
+    }
+
+    @Then("^The response should contain the data with the string value \"(.*?)\" and \"(.*?)\"$")
+    public void theResponseShouldContainSentDataWithStringValue(String tag, String tagValue) throws Throwable {
```
Comment (carledriss) [inline src/test/java/steps/ResponsesSteps.java]: Is it necessary throws Throwable ?

File: src/test/java/steps/ResourcesSteps.java (hunk 1 above)
Comment (carledriss) [inline src/test/java/steps/ResourcesSteps.java]: Can you apply the same fix for put request?

File: src/test/java/steps/ResponsesSteps.java (hunk 2 above)
Comment (robertoaban) [inline src/test/java/steps/ResponsesSteps.java]: is not necessary, is not been used, we are not catch any exception/error I will remove.

**Comment by carledriss** [comment]
Looks good to me!

**Pull request merged**